app/upgrades/v4/upgrade.go

```diff
@@ -112,6 +115,18 @@ func migrateRollappParams(ctx sdk.Context, rollappkeeper *rollappkeeper.Keeper)
 	rollappkeeper.SetParams(ctx, params)
 }
 
+// migrateRollappGauges creates a gauge for each rollapp in the store
+func migrateRollappGauges(ctx sdk.Context, rollappkeeper *rollappkeeper.Keeper, incentivizeKeeper *incentiveskeeper.Keeper) error {
+	rollapps := rollappkeeper.GetAllRollapps(ctx)
+	for _, rollapp := range rollapps {
```

Comment:
should we not skip frozen rollapps?

Reply:
I prefer not.
It doesn't really matter, as this gaguge won't be active for frozen rollapp, but it breaks the assumption of `rollapp gauge for each rollapp`